workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReaderBase.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using Improbable.Worker;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.MonoBehaviours
+{
+    public abstract class ReaderBase<TSpatialComponentData, TComponentUpdate>
+        : IReader<TSpatialComponentData>,
+            IReaderInternal
+        where TSpatialComponentData : ISpatialComponentData
+        where TComponentUpdate : ISpatialComponentUpdate<TSpatialComponentData>
+    {
+        protected readonly Unity.Entities.Entity Entity;
```

Comment:
Is there a way to get rid of the `Unity.Entities` prefix? Maybe something like `using Entity = Unity.Entities.Entity;`?